lib/AST/Decl.cpp

```diff
@@ -6814,6 +6814,14 @@ bool VarDecl::hasAttachedPropertyWrapper() const {
   return false;
 }
 
+/// Whether this property has any attached runtime metadata attributes.
+bool VarDecl::hasRuntimeMetadataAttributes() const {
+  if (!getRuntimeDiscoverableAttrs().empty())
```

Comment:
Nit: `return !getRuntimeDiscoverableAttrs().empty();` should do it :)

Reply:
😬 Thanks!